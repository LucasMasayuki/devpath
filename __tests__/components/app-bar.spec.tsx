import React from 'react'
import renderer from 'react-test-renderer'
import AppBar from '@/components/app-bar'

describe('AppBar', () => {
  it('should render AppBar', () => {
    const component = renderer.create(<AppBar />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
