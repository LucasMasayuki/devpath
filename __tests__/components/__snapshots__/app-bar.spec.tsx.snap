// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar should render AppBar 1`] = `
<div
  className="css-8eyg68"
>
  <div
    className="css-k008qs"
  >
    <div
      className="css-1o3848f"
    >
      <h2
        className="chakra-heading css-in3yi3"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          DEVMON
        </a>
      </h2>
    </div>
  </div>
</div>
`;
